src/Analysis/Engine/Impl/Values/BuiltinClassInfo.cs

```diff
@@ -127,19 +125,12 @@ public BuiltinClassInfo(IPythonType classObj, PythonAnalyzer projectState)
             return new BuiltinInstanceInfo(this);
         }
 
-        /// <summary>
-        /// Returns the overloads available for calling the constructor of the type.
-        /// </summary>
         public override IEnumerable<OverloadResult> Overloads {
             get {
-                // TODO: sometimes might have a specialized __init__.
-                // This just covers typical .NET types
-                var ctors = _type.GetConstructors();
-
-                if (ctors != null) {
-                    return ctors.Overloads.Select(ctor => new BuiltinFunctionOverloadResult(ProjectState, _type.Name, ctor, 1, () => Documentation));
-                }
-                return new OverloadResult[0];
+                var ctors = Type.GetConstructor();
```

Comment:
Let's leave the comment. We may need it during switching PTVS to LS

Reply:
Are we keeping IronPython in PTVS? It has been dead for many years.

Reply:
I'm not sure about it: https://github.com/IronLanguages/ironpython2/commits/master

Reply:
BTW, comment is misleading. This code is not .NET specific at all. It is primarily used in generating method override snippet in completion when deriving classes...